### PR TITLE
Adding TorchData tutorial

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #219
* #216
* __->__ #212

Adding a basic tutorial for TorchData

Differential Revision: [D34144164](https://our.internmc.facebook.com/intern/diff/D34144164)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34144164).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34144164).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34144164).

File: README.md [hunk 1]
```diff
@@ -251,11 +251,10 @@ For a complete example, suppose we want to load data from CSV files with the fol
 - Load csv files
 - Parse csv file and yield rows
 
-To support the above pipeline, `CSVParser` is registered as `parse_csv_files` to consume file streams and expand them as
-rows.
+To support the above pipeline, `CSVParser` is registered as `parse_csv` to consume file streams and expand them as rows.
 
 ```py
-@functional_datapipe("parse_csv_files")
+@functional_datapipe("parse_csv")
```
Comment (VitalyFedyunin) [inline README.md:257]: This could break TorchText datasets, can you please verify with them

Comment (NivekT) [reply README.md:257]: The functional API was changed a long time ago, it is the README file that didn't get updated.

The change here updates the README file, not the any code.

Comment (NivekT) [reply README.md:257]: I just checked that TorchText has updated their usage as well:

https://github.com/pytorch/text/search?q=parse_csv

Comment (VitalyFedyunin) [reply README.md:257]: Cool, just making sure we both fine in terms of releases and will not break someone.

**Comment by NivekT** [comment]
@ejguan @VitalyFedyunin Let me know if there is any suggestion for the tutorial here.

**Review by ejguan — APPROVED** [review]

**Pull request closed without merging**